overrides/runner.rb

```diff
@@ -23,76 +23,122 @@ module Overrides
   # description:
   #
   # overrides: !ruby/object:Overrides::ResourceOverrides
+  #   product: # Allows us to override information about the product.
+  #     prefix: magicproduct
   #   SomeResource: !ruby/object:Provider::MyProvider::ResourceOverride
   #     description: '{{description}} A tool-specific description complement'
-  #     parameters:
-  #       someParameter: !ruby/object:Provider::MyProvider::PropertyOverride
-  #         description: 'foobar' # replaces description
   #     properties:
-  #       someProperty: !ruby/object:Provider::MyProvider::PropertyOverride
+  #       someResourceRef: !ruby/object:Provider::MyProvider::PropertyOverride
+  #         type: Api::Type::Integer # changes type to an integer.
   #         description: 'foobar' # replaces description
   #       anotherProperty.someNestedProperty:
   #         !ruby/object:Provider::MyProvider::PropertyOverride
   #         description: 'baz'
-  #       anotherProperty[].someNestedPropertyInAnArray:
+  #       anotherProperty.someNestedPropertyInAnArray:
   #         !ruby/object:Provider::MyProvider::PropertyOverride
   #         description: 'baz'
   #   ...
   class Runner
-    class << self
-      def initialize(api, overrides,
-                     res_override_class = Overrides::ResourceOverride,
-                     prop_override_class = Overrides::PropertyOverride)
-        @api = api
-        @overrides = overrides
-        @res_override_class = res_override_class
-        @prop_override_class = prop_override_class
+    # Internal information for applying overrides.
```

Comment:
This doc string basically just repeats the name of the class; what kind of internal information, and how is it useful?